src/features/emails/layout/EmailLayout.tsx

```diff
@@ -1,41 +1,83 @@
-import { Box } from '@mui/material';
 import { FC } from 'react';
+import { Box, Typography } from '@mui/material';
 
 import EmailActionButtons from '../components/EmailActionButtons';
+import EmailStatusChip from '../components/EmailStatusChip';
 import messageIds from '../l10n/messageIds';
+import { People } from '@mui/icons-material';
 import TabbedLayout from '../../../utils/layout/TabbedLayout';
-import { useMessages } from 'core/i18n';
+import useEmail from '../hooks/useEmail';
+import useEmailState from '../hooks/useEmailState';
+// import useEmailStats from '../hooks/useEmailStats';
```

Comment:
If they're not used, please remove them.

Reply:
Sorry about that! I thought I would work on it so I left it but yeah I should remove it by now! Thanks for pointing it out!